pages/cloud/dedicated/partitioning_ovh/guide.en-gb.md

```diff
@@ -280,12 +280,12 @@ In order to improve customer experience, reduce [OVHcloud support](https://help.
 
 |Subject|Description|
 |---|---|
-|ZP grouping|All ZFS partitions with the same RAID level will be grouped within the same zpool (ZP) (if possible depending of the size of the disks)|
-|LV grouping|All partitions of type `lv` with the same RAID level will be grouped within the same VG (if possible depending of the size of the disks)|
+|ZP grouping|All ZFS partitions with the same RAID level will be grouped within the same zpool (ZP) (if possible depending on the size of the disks)|
+|LV grouping|All partitions of type `lv` with the same RAID level will be grouped within the same VG (if possible depending on the size of the disks)|
 |VG expanding|In case of LV partitions with a RAID level of 0, the VG will span multiple PPs (therefore PDs) and no SR device will be created|
 |VG Disk fill|Remaining disk space will be filled by a VG (if any LV exists). The size of LVs attached to the VG is not affected.|
-|RAID level reducing|In case the customer chose a partition with a RAID level that requires more disks than the server has, the RAID level will be automatically reduced in the following order: 6, 10, 5, 1, 0 (or raidz3, raidz2, raidz, mirror, striped vdev for ZFS)|
-|PP size reducing|In case the customer chose a PP that requires more space than the server has, the size of this PP will be reduced so that it fits the disk. Note that in case several PPs require more space than the system has, the script will only act on the first partition, raising an error later in the script for the second oversized partition. Also note that an error will be raised if the customer set another partition to fill the disk via the [OVHcloud API](https://api.ovh.com/)|
+|RAID level reducing|In case the customer choses a partition with a RAID level that requires more disks than the server has, the RAID level will be automatically reduced in the following order: 6, 10, 5, 1, 0 (or raidz3, raidz2, raidz, mirror, striped vdev for ZFS)|
```

Comment:
```suggestion
 |RAID level reducing|In case the customer chooses a partition with a RAID level that requires more disks than the server has, the RAID level will be automatically reduced in the following order: 6, 10, 5, 1, 0 (or raidz3, raidz2, raidz, mirror, striped vdev for ZFS)|
 ```